src/server/routes/routes.js

```diff
@@ -2,64 +2,37 @@ import express from 'express';
 
 import monthData from './monthData.js';
 import annualData from './annualData.js';
-import config from '../../../appConfig.js';
+import appConfig from '../../../appConfig.js';
 
 const router = express.Router();
-const { baseUrl } = config;
+const { baseUrl } = appConfig;
 
-/* monthOrAnnual
- * Checks whether the route params dictate either an annual selection or a monthly selection.
+/* getMonthData
  */
-function monthOrAnnual(req, res, next) {
-  const month = req.params.monthOrAnnual;
-  const id = req.params.idOrType;
-
-  if (month === 'books-music-dvds' && id === 'recommendations') {
-    return next();
-  }
-
-  if (month === 'annual') {
-    return annualData.selectAnnualData(req, res, next);
-  }
-
-  if (month === 'api') {
-    return next();
-  }
+function getMonthData(req, res, next) {
+  const month = req.params.month;
 
   if (!month.match(/[0-9]{4}-[0-9]{2}-[0-9]{2}/)) {
     return res.redirect(baseUrl);
   }
 
-  // There are more matches if anything is added after the url slug
-  if (id && id.match(/[0-9]*[-a-z]+/g).length > 1) {
-    return res.redirect(baseUrl);
-  }
-
   return monthData.selectMonthData(req, res, next);
 }
 
 router
-  .route('/')
```

Comment:
I think we need these routes for the reverse proxy case.

Reply:
We can add these back in later.

Reply:
Yes I agree, I think we should rethink the whole reverse proxy approach now that we have more AWS control over this and the gateway routing.